client/session/profile_manager_test.go

```diff
@@ -0,0 +1,341 @@
+package session_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/franela/goblin"
+	. "github.com/onsi/gomega"
+	"github.com/spacelift-io/spacectl/client/session"
```

Comment:
Nit: import grouping. At Spacelift, our practice is to split imports into up to 3 groups:

- stdlib imports;
- 3rd party imports;
- local imports;

Reply:
Before I go looking - do you guys just manually do that, or do you have your editors configured to do that automatically / can we lint it?

Reply:
I do it manually, but once I have 3 groups, `goimports` somehow "knows" where to put each new import. Maybe there's a way to automate that, haven't looked into it at all.